jobQueue/jobWrapper.js

```diff
@@ -265,6 +325,36 @@ function createPromisifiedJob(jobName, data, req) {
       });
   }
 
+  if (featureToggles.isFeatureEnabled('enableBullForBulkPostEvents') &&
+    jobName === jobType.bulkPostEvents) {
+    data.requestInfo = getRequestInfo(req);
+
+    const jobPriority = calculateJobPriority(conf.prioritizeJobsFrom,
+      conf.deprioritizeJobsFrom, req);
+
+    return bulkPostEventsQueue.add(data, { priority: jobPriority })
+      .then((job) => {
+        job.type = jobType.bulkPostEvents;
+        logJobCreate(startTime, job);
+        return job;
+      });
+  }
```

Comment:
If this is the same for each job, we should refactor to a separate function and pass in the job type.